test/e2e/data/infrastructure-vsphere-govmomi/v1.10/clusterclass/clusterclass-quick-start.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       - op: add
```

Comment:
Why do we have/need the additional base folder and commons/remove-storage-policy folder in v1.9?

(as far as I can tell they are unused in v1.9?)